test/utils.ts

```diff
@@ -30,11 +31,11 @@ const pks = [
 ]
 
 export function fromWei(amount: BN): string {
-  return web3.utils.fromWei(amount, "ether");
+  return Web3.utils.fromWei(amount, 'ether')
 }
 
 export function toWei(amount: string | number): BN {
```

Comment:
this should also accept a BigNumber, though if its working now as is then lets roll with it.